README.md

```diff
@@ -751,6 +751,37 @@ For supported properties, see
 [StreamClientConfiguration](https://fauna.github.io/fauna-js/latest/types/StreamClientConfiguration.html)
 in the API reference.
 
+## Debug logging
+
+To enable or disable debug logging, set the `FAUNA_DEBUG` environment variable
+to string-encoded
+[`LOG_LEVELS`](https://fauna.github.io/fauna-js/latest/variables/LOG_LEVELS.html)
```

Comment:
This 404s now, but it'll resolve with the next driver release. Can replace with a temp table if that's preferred.